static/app/components/onboarding/gettingStartedDoc/utils/profilingOnboarding.tsx

```diff
@@ -0,0 +1,12 @@
+import type {StepProps} from 'sentry/components/onboarding/gettingStartedDoc/step';
+
+export function getProfilingDocumentHeaderConfigurationStep(): StepProps {
+  return {
+    title: 'Add Document-Policy: js-profiling header',
+    description: `
```

Comment:
should `Document-Policy: js-profiling` be formatted as a code block for better legibility?

Reply:
Hmm, I'm not sure, but I think it would look weird if we had code in the title. Let me see if there is some better formatting here